pricing-page-exercise/saffet/styles.css

```diff
@@ -0,0 +1,257 @@
+body {
+    font-family: sans-serif;
+    margin: 0;
+    padding: 0;
+}
+
+
+/* Navigation bar*/
+.nav-bar{
+    box-sizing: border-box;
```

Comment:
Always add box-sizing at the very beginning of the main CSS file to all elements at once, otherwise it will be very hard to debug unexpected behaviour. 
```
* { 
      box-sizing: border-box;
}
```